app/AcquireIntersonArrayRF.cxx

```diff
@@ -121,18 +121,18 @@ int main( int argc, char * argv[] )
   ContainerType container;
 
   const int height = hwControls.GetLinesPerArray();
+  std::cout << "Lines per array: " << height << std::endl;
   const int width = ContainerType::MAX_RFSAMPLES;
```

Comment:
Swapping the MAX_RFSamples and NBOFLINES at line 344 in the other file results in an array index out of bounds.   In the example code, the RF arrays are initialized (rows=NBOFLINES, cols=MAX_RF).

Swapping # of lines and max_samples here results in an incorrect image (usual row/column mismatch artifacts)